common/app/views/fragments/nav/newHeaderMenu.scala.html

```diff
@@ -174,6 +174,14 @@
                     </a>
                 </li>
             </ul>
+
+            <div class="hide-until-desktop feedback-form">
+                @fragments.inlineSvg("information-circle", "icon", List("feedback-form__icon"))
+                This navigation is new. We'd appreciate your feedback.
+                <a class="feedback-form__link"
+                    href="https://docs.google.com/a/guardian.co.uk/forms/d/e/1FAIpQLSc0Zv6z5TJp_GoedDVt0Z6X7o41fqB3nniKHRaxah0VRGce4w/viewform">
+                    Share your thoughts here.</a>
```

Comment:
Oh, I've overseen this yesterday: I'd propose to either convert the wrapping div into a `<p>` or change the link markup to

```html
<a class="feedback-form__link"
+                    href="https://docs.google.com/a/guardian.co.uk/forms/d/e/1FAIpQLSc0Zv6z5TJp_GoedDVt0Z6X7o41fqB3nniKHRaxah0VRGce4w/viewform">
+                    Share your <span class="u-h">Feedback</span> thoughts here.</a>
```

for accessibility. Prefer the first proposal though.